### PR TITLE
DDCE-1711 Documentation updates

### PR DESCRIPTION
**Pull request by Taiwo-AB**

### CODE REVIEW

File: resources/public/api/conf/1.0/application.yaml [hunk 1]
```diff
@@ -34,26 +37,39 @@ paths:
...
-                  description: Example response for scenario HAPPY_PATH_2
-                  value:
-                    class1:
-                      totalNICableEarnings: 17
-                    class2:
-                      totalDue: 319.57
-                    maxNICsReached: true
+                happy-path-1:
+                  $ref: '#/components/examples/responseHappyPath1'
+                happy-path-2:
```
Comment (gastonracui) [inline resources/public/api/conf/1.0/application.yaml:42]: there is no happy path 2 for NI

File: resources/public/api/conf/1.0/application.yaml [hunk 2]
```diff
@@ -22,8 +22,11 @@ paths:
             schema:
               allOf:
                 - $ref: '#/components/schemas/nationalInsuranceTestSupportSaAnnualSummaryRequest'
-            example:
-              scenario: HAPPY_PATH_1
+            examples:
+              happy-path-1:
+                $ref: '#/components/examples/requestHappyPath1'
+              happy-path-2:
+                $ref: '#/components/examples/requestHappyPath2'
```
Comment (gastonracui) [inline resources/public/api/conf/1.0/application.yaml:29]: no PATH2 for NI

**Review by gastonracui — CHANGES_REQUESTED** [review]

File: resources/public/api/conf/1.0/application.yaml (hunk 1 above)
Comment (Taiwo-AB) [reply resources/public/api/conf/1.0/application.yaml:42]: There is. See [here](https://github.com/hmrc/national-insurance-des-stub/blob/main/resources/public/scenarios/HAPPY_PATH_2.json) and the screenshot. 

happy path 2

File: resources/public/api/conf/1.0/application.yaml (hunk 2 above)
Comment (Taiwo-AB) [reply resources/public/api/conf/1.0/application.yaml:29]: See above

**Pull request merged**